### PR TITLE
WIP: Build homepage

### LINKED ISSUE #7970: Build ubuntu.com homepage
Build the ubuntu.com homepage to the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/7969) and [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit).

### PR DESCRIPTION
**Pull request by MeltySnow**
## Done

- Build the homepage based on the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit) and [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/7969)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the content matches the copy doc
- Ensure that the code is sensible and in line with any Canonical style guides
- Check that the layout of the page is suitable - there are several instances where Vanilla's default behaviour did not give 
a 1:1 correspondence with the design

## Issue / Card

Fixes #7970

### CODE REVIEW

**Comment by meralonso** [comment]
@MeltySnow would it be possible to have 6 logos per row instead of 5? except for the ones at the top in b/w which is fine like that as they are 10. 

@pmahnke are we still good to go for 6 logos per row and get rid of the rest?

File: templates/base_index.html [hunk 1]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">No mandatory subscriptions for Ubuntu</li>
+          <li class="is-ticked">Security, support and fully managed offerings</li>
+          <li class="is-ticked">Public cloud, data centre, edge cluster & appliances</li>
+          <li class="is-ticked">Long term maintenance commitment</li>
+          <li class="is-ticked">Transparent pricing for enterprise and ISV</li>
+          <li class="is-ticked"><a href="/engage/vmware-to-charmed-openstack">Switch from VMware to OpenStack</a></li>
+          <li class="is-ticked"><a href="/tco-calculator">TCO analysis of Open Infra</a></li>
+          <li class="is-ticked"><a href="/engage/cloud-economics">451 study on private/public cloud costs</a></li>
+        </ul>
+        <a href="#get-in-touch">Let’s talk open source ></a>
```
Comment (anthonydillon) [inline templates/base_index.html:315]:
```suggestion
        <a href="/contact-us" class="js-invoke-modal">Let’s talk open source&nbsp;&rsaquo;</a>
```

Comment (anthonydillon) [reply templates/base_index.html:315]: The reason for this is that `#get-in-touch` requires JS to trigger the modal contact form so if someone doesn't have js they will link to the contact-us page but if they do the modal will takeover.

File: templates/base_index.html [hunk 2]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+      </div>
+    </section>
+
+    <section class="p-strip">
+      <div class="u-fixed-width">
+        <h2>Public cloud optimisation</h2>
+        <h5>Most public cloud instances are Ubuntu, for performance and security</h5>
```
Comment (anthonydillon) [inline templates/base_index.html:322]: p with p-heading--five, please

File: templates/base_index.html [hunk 3]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        <ul class="p-list is-split">
+          <li class="is-ticked">Optimised kernels for AWS, Azure, Google, Oracle, IBM</li>
```
Comment (anthonydillon) [inline templates/base_index.html:324]: These need `p-list__item`

File: templates/base_index.html [hunk 4]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">GPU enablement for machines and containers</li>
+          <li class="is-ticked">Pay-as-you-go billing, no more licenses</li>
+          <li class="is-ticked">Enterprise support and Extended Security Maintenance</li>
+          <li class="is-ticked">Cloud workload migration service</li>
+          <li class="is-ticked">Containerisation and cloud-native migration service</li>
+          <li class="is-ticked">Fully managed application service</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/public-cloud">Learn more ></a></li>
+          <li class="p-inline-list__item"><a href="/public-cloud#get-in-touch">Contact us ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:337]:
```suggestion
          <li class="p-inline-list__item"><a href="/public-cloud/contact-us">Contact us&nbsp;&rsaquo;</a></li>
```

File: templates/base_index.html [hunk 5]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">Network & storage performance optimisation</li>
+          <li class="is-ticked">GPU enablement for machines and containers</li>
+          <li class="is-ticked">Pay-as-you-go billing, no more licenses</li>
+          <li class="is-ticked">Enterprise support and Extended Security Maintenance</li>
+          <li class="is-ticked">Cloud workload migration service</li>
+          <li class="is-ticked">Containerisation and cloud-native migration service</li>
+          <li class="is-ticked">Fully managed application service</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/public-cloud">Learn more ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:336]: Please encode with `&nbsp;&rsaquo;`

File: templates/base_index.html [hunk 6]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        <h2>Carrier-grade private cloud</h2>
+        <h5>Operate <a href="/openstack">private cloud the smart way</a> - supported or <a href="/openstack/managed">fully managed</a></h5>
```
Comment (anthonydillon) [inline templates/base_index.html:395]:
```suggestion
        <p class="p-heading--five">Operate <a href="/openstack">private cloud the smart way</a> &mdash; supported or <a href="/openstack/managed">fully managed</a></p>
```

File: templates/base_index.html [hunk 7]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        <ul class="p-list is-split">
+          <li class="is-ticked">10 year security maintenance and support</li>
```
Comment (anthonydillon) [inline templates/base_index.html:397]: These all need `p-list__item`

File: templates/base_index.html [hunk 8]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip">
+      <div class="u-fixed-width">
+        <h2>Multi-cloud Kubernetes and containers</h2>
+        <h5>AKS. EKS. GKE. Kubeadm. MicroK8s. Charmed Kubernetes. All on Ubuntu.</h5>
+        <ul class="p-list is-split">
+          <li class="is-ticked">Support for all public cloud Kubernetes</li>
```
Comment (anthonydillon) [inline templates/base_index.html:505]: These all need `p-list__item`

File: templates/base_index.html [hunk 9]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip">
+      <div class="u-fixed-width">
+        <h2>Multi-cloud Kubernetes and containers</h2>
+        <h5>AKS. EKS. GKE. Kubeadm. MicroK8s. Charmed Kubernetes. All on Ubuntu.</h5>
```
Comment (anthonydillon) [inline templates/base_index.html:503]: p with p-heading--five, please

File: templates/base_index.html [hunk 10]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">Full stack support for apps, Kubernetes, and OS</li>
+          <li class="is-ticked">K8s architecture and deployment consulting</li>
+          <li class="is-ticked">Cloud-native workload migration services</li>
+          <li class="is-ticked">GPU enablement for AI/ML workloads</li>
+          <li class="is-ticked">Bare metal, OpenStack and VMware</li>
+          <li class="is-ticked">Pluggable CNIs, CSIs, CRIs and third-party tools</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/kubernetes">Learn more ></a></li>
+          <li class="p-inline-list__item"><a href="/kubernetes/#get-in-touch">Contact us ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:518]: Please encode the > and link to the contact-us page not the 'get-in-touch page.

File: templates/base_index.html [hunk 11]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        <h2>Ultra secure things</h2>
+        <h5><a href="/appliance">Ubuntu Core Appliances</a> with transactional updates <a href="/embedded">for a better embedded Linux</a></h5>
```
Comment (anthonydillon) [inline templates/base_index.html:600]: p with p-heading--five, please

File: templates/base_index.html [hunk 12]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        <ul class="p-list is-split">
+          <li class="is-ticked">OTA compressed updates for apps, kernel and OS</li>
```
Comment (anthonydillon) [inline templates/base_index.html:602]: Need `p-list__item` class

File: templates/base_index.html [hunk 13]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">Built-in device recovery and remote disk repair</li>
+          <li class="is-ticked">10 year security maintenance for every device</li>
+          <li class="is-ticked">Provable system and software integrity</li>
+          <li class="is-ticked">Secure boot backed by root of trust</li>
+          <li class="is-ticked">Full disk encryption with hardware keys</li>
+          <li class="is-ticked">FIPS certifiable embedded Linux</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/core">Learn more ></a></li>
+          <li class="p-inline-list__item"><a href="/core#get-in-touch">Contact us ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:615]: Please encode the > and link to the contact-us page not the #get-in-touch

File: templates/base_index.html [hunk 14]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip">
+      <div class="u-fixed-width">
+        <h2>Workstations and desktops</h2>
+        <h5>Optimised AI/ML and cloud devops. The professional developer’s preference.</h5>
```
Comment (anthonydillon) [inline templates/base_index.html:685]: p with p-heading--five, please

File: templates/base_index.html [hunk 15]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        <ul class="p-list is-split">
+          <li class="is-ticked">Active Directory and LDAP integration</li>
```
Comment (anthonydillon) [inline templates/base_index.html:687]: Need `p-list__item` class

File: templates/base_index.html [hunk 16]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">Security updates for the full open source stack</li>
+          <li class="is-ticked">Estate monitoring and management</li>
+          <li class="is-ticked">Configuration lockdown</li>
+          <li class="is-ticked">Local Kubernetes for offline development</li>
+          <li class="is-ticked">Instant Ubuntu on Windows or macOS with WSL and Multipass</li>
+          <li class="is-ticked">Optimised images and certified workstations</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/desktop">Learn more ></a></li>
+          <li class="p-inline-list__item"><a href="/desktop/organisations#get-in-touch">Contact us ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:698]: Please encode the > and link to the contact-us page not the #get-in-touch

File: templates/base_index.html [hunk 17]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        <h2>Data centre automation</h2>
+        <h5>Turn a rack into a bare metal cloud. <a href="https://maas.io/" class="p-link--external">Metal-as-a-Service provisioning</a> of Windows, ESX, Linux.</h5>
```
Comment (anthonydillon) [inline templates/base_index.html:768]: p with p-heading--five, please

File: templates/base_index.html [hunk 18]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        <ul class="p-list is-split">
+          <li class="is-ticked">Bare metal cloud with on-demand servers</li>
```
Comment (anthonydillon) [inline templates/base_index.html:770]: Need `p-list__item` class

File: templates/base_index.html [hunk 19]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">Super fast install from scratch</li>
+          <li class="is-ticked">VMWare ESX, Windows, CentOS, RHEL, Ubuntu</li>
+          <li class="is-ticked">Custom images with pre-installed apps</li>
+          <li class="is-ticked">Disk and network configuration</li>
+          <li class="is-ticked">API-driven DHCP, DNS, PXE, IPAM, provisioning</li>
+          <li class="is-ticked">LDAP authentication and RBAC</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="https://maas.io/" class="p-link--external">Learn more ></a></li>
+          <li class="p-inline-list__item"><a href="https://maas.io#get-in-touch" class="p-link--external">Contact us ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:783]: Please encode the > and link to the contact-us page not the #get-in-touch

File: templates/base_index.html [hunk 20]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip">
+      <div class="u-fixed-width">
+        <h2>Smart robots of all shapes and sizes</h2>
+        <h5>From turtles to trucks, <a href="/robotics">Ubuntu drives the robot revolution</a></h5>
```
Comment (anthonydillon) [inline templates/base_index.html:877]: p with the p-heading--five class, please

File: templates/base_index.html [hunk 21]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+        <ul class="p-list is-split">
+          <li class="is-ticked">Full support for ROS and ROS2</li>
```
Comment (anthonydillon) [inline templates/base_index.html:879]: Missing the p-list__item class

File: templates/base_index.html [hunk 22]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="is-ticked">Security patches for long term ROS</li>
+          <li class="is-ticked">Fault tolerant over-the-air updates</li>
+          <li class="is-ticked">CIS certification for ROS and ROS2</li>
+          <li class="is-ticked">Trim time-to-market with SMART START consulting</li>
+          <li class="is-ticked">Leading Robot-as-a-Platform ecosystem</li>
+          <li class="is-ticked">Curated app stores</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/robotics">Learn more ></a></li>
+          <li class="p-inline-list__item"><a href="/robotics#get-in-touch">Contact us ></a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:889]: Please encode the > and link to the contact-us page not the #get-in-touch

File: templates/base_index.html [hunk 23]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="row">
+        <div class="col-8">
+          <h2>Multi-cloud applications - beyond PAAS</h2>
```
Comment (anthonydillon) [inline templates/base_index.html:995]: Encode the - with &ndash; (tip: ndash for headings, mdash for body copy)

File: templates/base_index.html [hunk 24]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <h5>Universal model-driven operators for classic and Kubernetes estate</h5>
+          <ul class="p-list">
+            <li class="is-ticked">Operate smoothly across public and private cloud</li>
```
Comment (anthonydillon) [inline templates/base_index.html:998]: Need p-list__item class, please

File: templates/base_index.html [hunk 25]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            <li class="is-ticked">Reusable integration code as open source packages</li>
+            <li class="is-ticked">Python Operator Framework for rapid development</li>
+            <li class="is-ticked">Bring the K8s operator pattern to classic workloads</li>
+            <li class="is-ticked">Lifecycle management and Day 2 operations</li>
+            <li class="is-ticked">Open collection of community-driven operators</li>
+          </ul>
+          <a href="https://juju.is/">Learn more ></a>
```
Comment (anthonydillon) [inline templates/base_index.html:1005]: Please encode the > and add an external link class

**Comment by anthonydillon** [comment]
## QA
- The design has the hackerearch logo above the link:
image

- This section does not match the copy doc:
image
I would suggest replacing that strip with (of course the image should be the image template with the height and width adjusted):
```html
<section class="p-strip--light">
  <div class="row u-equal-height">
    <div class="col-7">
        <h2>Significant enterprise savings</h2>
        <h5>Sustainable open source at the lowest cost, everywhere</h5>
        <p>Rethink what’s possible with Linux and open source.</p>
        <p>Companies engage Canonical to drive down open source operating costs. Automate everything: multi-cloud operations, bare metal provisioning, edge clusters and IoT.</p>
        <ul class="p-list">
          <li class="p-list__item is-ticked">No mandatory subscriptions for Ubuntu</li>
          <li class="p-list__item is-ticked">Security, support and fully managed offerings</li>
          <li class="p-list__item is-ticked">Public cloud, data centre, edge cluster &amp; appliances</li>
          <li class="p-list__item is-ticked">Long term maintenance commitment</li>
          <li class="p-list__item is-ticked">Transparent pricing for enterprise and ISV</li>
          <li class="p-list__item is-ticked"><a href="/engage/vmware-to-charmed-openstack">Switch from VMware to OpenStack</a></li>
          <li class="p-list__item is-ticked"><a href="/tco-calculator">TCO analysis of Open Infra</a></li>
          <li class="p-list__item is-ticked"><a href="/engage/cloud-economics">451 study on private/public cloud costs</a></li>
        </ul>
        <a href="/contact-us" class="js-invoke-modal">Let’s talk open source&nbsp;›</a>
    </div>
    <div class="col-5 u-vertically-center u-hide--small u-align--center">
      <div class=" lazyloaded" data-noscript="">
        <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_128,h_128/https://assets.ubuntu.com/v1/3ec2dd61-Money.svg" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_256,h_256/https://assets.ubuntu.com/v1/3ec2dd61-Money.svg 2x" alt="" loading="lazy" width="250" height="250">
      </div>
    </div>
  </div>
</section>
```

-

**Comment by pmahnke** [comment]
@anthonydillon the 'The design has the hackerearch logo above the link' I don't think is something we do, I would personally leave it.

**Comment by meralonso** [comment]
> @anthonydillon the 'The design has the hackerearch logo above the link' I don't think is something we do, I would personally leave it.

@pmahnke @anthonydillon I'm fine with the text above or under the logo, but if you could left align them that'll be great. 
Also, any thoughts on my question about having 6 logos per row instead of 5?

**Comment by anthonydillon** [comment]
@pmahnke @meralonso @MeltySnow Fine by me if the design has changed. I was just using the design to match it up :+1:

**Comment by MeltySnow** [comment]
I've pushed up changes that should address @anthonydillon's comments relating to the code and @meralonso's suggestion to make the HackerEarth logo align left.

With respect to @meralonso's question about having 6 logos per row, Vanilla automatically wraps it at 5 by default. It would be possible to hack, perhaps by using multiple `p-inline-list--stretch` in place of `p-inline-images`, but as hacking or custom styling are discouraged I'd like approval from @anthonydillon or @pmahnke before implementing something like that.

File: templates/base_index.html [hunk 26]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+
+    <section class="p-strip">
+      <div class="u-fixed-width">
+        <h2>Public cloud optimisation</h2>
+        <p class="p-heading--five">Most public cloud instances are Ubuntu, for performance and security</p>
+        <ul class="p-list is-split">
+          <li class="p-list__item is-ticked">Optimised kernels for AWS, Azure, Google, Oracle, IBM</li>
+          <li class="p-list__item is-ticked">FIPS, CIS hardening and CVE fixes with Ubuntu Pro</li>
+          <li class="p-list__item is-ticked">Minimise rolling reboots with Kernel Livepatch</li>
+          <li class="p-list__item is-ticked">Network & storage performance optimisation</li>
```
Comment (anthonydillon) [inline templates/base_index.html:327]: Sorry the & should be encoded with `&amp;`

File: templates/base_index.html [hunk 27]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="p-list__item is-ticked">Built-in device recovery and remote disk repair</li>
+          <li class="p-list__item is-ticked">10 year security maintenance for every device</li>
+          <li class="p-list__item is-ticked">Provable system and software integrity</li>
+          <li class="p-list__item is-ticked">Secure boot backed by root of trust</li>
+          <li class="p-list__item is-ticked">Full disk encryption with hardware keys</li>
+          <li class="p-list__item is-ticked">FIPS certifiable embedded Linux</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/core">Learn more&nbsp;&rsaquo;</a></li>
+          <li class="p-inline-list__item"><a href="/contact-us" class="js-invoke-modal">Contact us&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:615]: Shouldn't this link go to `/core/contact-us`

File: templates/base_index.html [hunk 28]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="p-list__item is-ticked">Security updates for the full open source stack</li>
+          <li class="p-list__item is-ticked">Estate monitoring and management</li>
+          <li class="p-list__item is-ticked">Configuration lockdown</li>
+          <li class="p-list__item is-ticked">Local Kubernetes for offline development</li>
+          <li class="p-list__item is-ticked">Instant Ubuntu on Windows or macOS with WSL and Multipass</li>
+          <li class="p-list__item is-ticked">Optimised images and certified workstations</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/desktop">Learn more&nbsp;&rsaquo;</a></li>
+          <li class="p-inline-list__item"><a href="/contact-us" class="js-invoke-modal">Contact us&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:698]: /desktop/contact-us ?

File: templates/base_index.html [hunk 29]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="p-list__item is-ticked">Wide choice of SDN and storage</li>
+          <li class="p-list__item is-ticked">Upgrades and day-2 operations, automated</li>
+          <li class="p-list__item is-ticked">Integrated logging, monitoring and alerting</li>
+          <li class="p-list__item is-ticked">Platform and infrastructure hardening</li>
+          <li class="p-list__item is-ticked">High availability</li>
+          <li class="p-list__item is-ticked">Carrier-grade NFVI</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/openstack">Learn more&nbsp;&rsaquo;</a></li>
+          <li class="p-inline-list__item"><a href="/contact-us" class="js-invoke-modal">Contact us&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:409]: /openstack/contact-us ?

File: templates/base_index.html [hunk 30]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <li class="p-list__item is-ticked">Super fast install from scratch</li>
+          <li class="p-list__item is-ticked">VMWare ESX, Windows, CentOS, RHEL, Ubuntu</li>
+          <li class="p-list__item is-ticked">Custom images with pre-installed apps</li>
+          <li class="p-list__item is-ticked">Disk and network configuration</li>
+          <li class="p-list__item is-ticked">API-driven DHCP, DNS, PXE, IPAM, provisioning</li>
+          <li class="p-list__item is-ticked">LDAP authentication and RBAC</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="https://maas.io/" class="p-link--external">Learn more</a></li>
+          <li class="p-inline-list__item"><a href="/contact-us" class="p-link--external js-invoke-modal">Contact us</a></li>
```
Comment (anthonydillon) [inline templates/base_index.html:783]: MAAS has its own contact form https://maas.io/contact-us but if you dont want to use it you should remove the external class and add the chevron.

File: templates/base_index.html [hunk 31]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        <h2>Ultra secure things</h2>
+        <p class="p-heading--five"><a href="/appliance">Ubuntu Core Appliances</a> with transactional updates <a href="/embedded">for a better embedded Linux</a></p>
```
Comment (anthonydillon) [inline templates/base_index.html:600]: This sentence should have a full stop at the end.

File: templates/base_index.html [hunk 32]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        <h2>Carrier-grade private cloud</h2>
+        <p class="p-heading--five">Operate <a href="/openstack">private cloud the smart way</a>&nbsp;&ndash;&nbsp;supported or <a href="/openstack/managed">fully managed</a></p>
```
Comment (anthonydillon) [inline templates/base_index.html:395]: Can you add a full stop at the end please.

**Comment by MeltySnow** [comment]
@anthonydillon Done! Sorry about the contact-us links; I must've seen that /public-cloud/contact-us was a 404 and assumed that all the others were too.

**Comment by meralonso** [comment]
> I've pushed up changes that should address @anthonydillon's comments relating to the code and @meralonso's suggestion to make the HackerEarth logo align left.
> 
> With respect to @meralonso's question about having 6 logos per row, Vanilla automatically wraps it at 5 by default. It would be possible to hack, perhaps by using multiple `p-inline-list--stretch` in place of `p-inline-images`, but as hacking or custom styling are discouraged I'd like approval from @anthonydillon or @pmahnke before implementing something like that.

@pmahnke should we just go for 5 logos max per row then?

File: templates/base_index.html [hunk 33]
```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
...
+          <p class="p-heading--five">Universal model-driven operators for classic and Kubernetes estate.</p>
+          <ul class="p-list">
+            <li class="p-list__item is-ticked">Operate smoothly across public and private cloud</li>
+            <li class="p-list__item is-ticked">Reusable integration code as open source packages</li>
+            <li class="p-list__item is-ticked">Python Operator Framework for rapid development</li>
+            <li class="p-list__item is-ticked">Bring the K8s operator pattern to classic workloads</li>
+            <li class="p-list__item is-ticked">Lifecycle management and Day 2 operations</li>
+            <li class="p-list__item is-ticked">Open collection of community-driven operators</li>
+          </ul>
+          <a href="https://juju.is/" class="p-link--external">Learn more</a>
```
Comment (anthonydillon) [inline templates/base_index.html:1005]:
```suggestion
          <p><a href="https://juju.is/" class="p-link--external">Learn more</a></p>
```

**Comment by MeltySnow** [comment]
@meralonso I've just asked Peter and he said he's waiting to hear back from product management on the issue of 5 vs 6 logos per row, and also on the possibility of removing some logos.

**Comment by MeltySnow** [comment]
@anthonydillon Done!

**Comment by anthonydillon** [comment]
@MeltySnow nice work. I have run out of things to complain about! :+1:

**Comment by meralonso** [comment]
@MeltySnow just saw this logo cloud on one of the [OpenStack](https://ubuntu.com/openstack/what-is-openstack) pages - if we can do something similar that'll be great. Using 5 or 6 logos and all the way to the edges. 

Screenshot 2020-08-12 at 15 45 39

**Comment by MeltySnow** [comment]
@meralonso Thank you for the suggestion; for the most part it worked well and I've pushed up a change.

However, getting the rows with 5 logos to stretch the full length of the page proved to be slightly tricky, as the 12 available columns are not evenly divisible by 5.

I tried using `p-inline-list--stretch`, but this resulted in noticeable vertical offset between the images:
Screenshot 2020-08-12 at 17 49 07

and applying `u-vertically-center` seems to break the inline list:
Screenshot 2020-08-12 at 17 49 56

So for now I've just tried to centre the list by adding an empty `<div class="col-1">` before the first `<li>`.
Screenshot 2020-08-12 at 17 45 48

If anyone has any better ideas for how to do this, I am open to suggestions.

**Comment by pmahnke** [comment]
@meralonso @MeltySnow I am a bit hesitant to not use the normal Vanilla pattern for logos.  I don't mind aligning them left, but I don't really even like going into col-2s - I feel like we should fix Vanilla first.  And creating a new col-1/5 seems wrong.  

@bartaz and/or @anthonydillon what do you think the best thing to do is?

**Comment by anthonydillon** [comment]
Absolutely, we should use Vanilla first and foremost. Extensions can be done locally but this feels like a modification to the inline-image pattern in Vanilla.

**Comment by meralonso** [comment]
> @meralonso @MeltySnow I am a bit hesitant to not use the normal Vanilla pattern for logos. I don't mind aligning them left, but I don't really even like going into col-2s - I feel like we should fix Vanilla first. And creating a new col-1/5 seems wrong.
> 
> @bartaz and/or @anthonydillon what do you think the best thing to do is?

@MeltySnow ok let's leave it as it is for now, but if we could do 6 logos per row @pmahnke maybe that'll fix the problem (visually, I mean)

**Comment by anthonydillon** [comment]
Superseeded by #8173

**Pull request closed without merging**